lib/author.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/book'
```

Comment:
This is only needed in the test file since you're running everything from there.